src/lib/utilities/decode-payload.test.ts

```diff
@@ -106,6 +116,9 @@ describe('decodePayload', () => {
   it('Should decode a json payload with encoding json/plain', () => {
     expect(decodePayload(JsonObjectEncoded)).toEqual(JsonObjectDecoded);
   });
+  it('Should decode a json payload with constructor keyworkd with encoding json/plain', () => {
```

Comment:
Should this be `json/protobuf` ? I thought that's the specific encoding that caused the bug. Or was it the same for all encodings?

Reply:
We have tests for json/protobuf. We decode anything that is json/....